pro/src/components/OfferEducationalActions/__specs__/OfferEducationalActions.spec.tsx

```diff
@@ -110,4 +114,51 @@ describe('OfferEducationalActions', () => {
       }
     )
   })
+  it('should display error message when trying to activate offer with booking limit date time in the past', async () => {
+    const notifyError = jest.fn()
+    jest.spyOn(useNotification, 'default').mockImplementation(() => ({
+      ...jest.requireActual('hooks/useNotification'),
+      error: notifyError,
+    }))
+    renderOfferEducationalActions({
+      ...defaultValues,
+      isOfferActive: false,
+      offer: collectiveOfferFactory({
+        isActive: false,
+        collectiveStock: collectiveStockFactory({
+          bookingLimitDatetime: '1900-10-15T00:00:00Z',
+        }),
+      }),
+    })
+    await userEvent.click(
+      screen.getByRole('button', { name: 'Publier sur Adage' })
+    )
+    expect(defaultValues.setIsOfferActive).not.toHaveBeenCalled()
+    expect(notifyError).toHaveBeenCalledWith(
+      'La date limite de réservation est dépassée. Pour publier l’offre, vous devez modifier la date limite de réservation.'
+    )
+  })
+  it('should activate offer with booking limit date time in the future', async () => {
```

Comment:
(détail) même si prettier le fait pas auto j'aime bien les lignes vide entre les `it` pour aérer :)